android-slack/src/main/java/com/genymobile/android_slack/impl/Message.java

```diff
@@ -0,0 +1,52 @@
+package com.genymobile.android_slack.impl;
```

Comment:
Or `internal` ? this is for implementation details of the library, it is not the public API